opennlp-tools/src/test/java/opennlp/tools/lemmatizer/DummyLemmatizer.java

```diff
@@ -56,4 +57,10 @@ public DummyLemmatizer(DummyLemmaSampleStream aSampleStream) {
     }
   }
 
+  @Override
+  public List<List<String>> lemmatize(List<String> toks,
+      List<String> tags) {
+    return null;
```

Comment:
Sorry if I my question wasn't clear. I was wondering if it was meant it to be `return null` or if it was just forgotten.